app/src/lib/shells/darwin.ts

```diff
@@ -36,6 +41,8 @@ function getBundleID(shell: Shell): string {
       return 'com.googlecode.iterm2'
     case Shell.Hyper:
       return 'co.zeit.hyper'
+    case Shell.PowerShellCore:
+      return 'com.microsoft.powershell'
```

Comment:
Minor thing: I tested this with the [binary installation](https://github.com/PowerShell/PowerShell/blob/master/docs/installation/macos.md#installation-via-binary-archive) steps and Desktop didn't detect it. `brew cask install powershell` worked fine, and I gather the direct install option also works.

I'm not sure how many people would install it manually _as well as want to use it from Desktop_, but if we have to support that option we might have to fall back to how Linux resolves it's shells. But I won't hold up this PR for that.